### ISSUE TITLE
DevTools RFC team link is broken

### ISSUE THREAD

**Issue by past** [firefox-devtools/rfcs#64]
In the Readme file the DevTools RFC team link is 404.

**Comment by zoepage** [comment]
I checked all links in https://github.com/firefox-devtools/rfcs/blob/master/README.md and they work. 
We you referring to this file?

**Comment by past** [comment]
Yes, specifically the [first link](https://github.com/orgs/firefox-devtools/teams/devtools-rfc/members). It may be a permissions issue then. Does it work for you in private browsing mode or without logging in?

**Comment by zoepage** [comment]
In a private window, I am forwarded to the login page. I also assume it is a permission issue with the visibility of the team set to private.
